benchmark/execute-overhead.ts

```diff
@@ -3,42 +3,38 @@
 
 import * as napa from '../lib/index';
 import * as mdTable from 'markdown-table';
-import { formatTimeDiff } from './bench-utils';
+import {formatTimeDiff} from './bench-utils';
+
+function batchExecuteOnNamedFunction(zone: napa.zone.Zone, repeat: number, args: any[]): Promise<void> {
```

Comment:
The column limit.

Current clang-format config is simple:
1. when it can be put in one line, then do it
2. otherwise, when it can be put in one line in the second line with indent, then do it
3. otherwise, one line per item